pkg/controller/sync/controller.go

```diff
@@ -293,6 +293,7 @@ func (s *KubeFedSyncController) syncToClusters(fedResource FederatedResource) ut
 	// Enable raw resource status collection if the statusCollection is enabled for that type
 	// and the feature is also enabled.
 	enableRawResourceStatusCollection := s.typeConfig.GetStatusEnabled() && s.rawResourceStatusCollection
+	klog.V(4).Infof("typeconfigstatus for %v enabled: %v, rawresstatus: %v", fedResource.FederatedName(), s.typeConfig.GetStatusEnabled(), enableRawResourceStatusCollection)
```

Comment:
I'll rephrase this entry to `"federated status for '%q' enabled '%b' with resource status collection enabled: '%b'"`